### ISSUE TITLE
Make 4.4 cannot be fetched

### ISSUE THREAD

**Issue by TheGrizzlyDev** [bazelbuild/rules_foreign_cc#982]
As of commit [78acc733fb2e75236fbe48237ae19ea740767aa9](https://github.com/bazelbuild/rules_foreign_cc/commit/78acc733fb2e75236fbe48237ae19ea740767aa9) `rules_foreign_cc_dependencies` will prefer fetching 4.4 as its default make toolchain. Sadly, [bazel's mirror URL](https://mirror.bazel.build/ftpmirror.gnu.org/gnu/make/make-4.4.tar.gz) returns 404.

**Comment by jsharpe** [comment]
That's not something I can fix - it relies on a googler to update the mirror with such packages.
Is this causing a failure for you? It should fetch from the gnu mirror when the 404 on the mirror is found.

**Comment by TheGrizzlyDev** [comment]
No, I can also just set it to use 4.3 until it's fixed, no worries. I would prefer though if the bazel mirror was added only after actually uploading the file which would cause less confusion on the end user. What do you think?

**Comment by jsharpe** [comment]
@Wyverald is there a process for getting packages mirrored onto mirror.bazel.build? I know you've setup (or are planning on) some automation for the BCR, is there something equivalent for mirror.bazel.build?

**Comment by Wyverald** [comment]
you can file an issue (https://github.com/bazelbuild/bazel/issues/new/choose) using the "mirror request" template

**Comment by jsharpe** [comment]
Filed: https://github.com/bazelbuild/bazel/issues/16676

**Comment by TheGrizzlyDev** [comment]
Thanks!
